[2024-06-22T19:20:34.655+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-22T19:15:02.330472+00:00 [queued]>
[2024-06-22T19:20:34.660+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-22T19:15:02.330472+00:00 [queued]>
[2024-06-22T19:20:34.661+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-06-22T19:20:34.668+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2024-06-22 19:15:02.330472+00:00
[2024-06-22T19:20:34.672+0000] {standard_task_runner.py:57} INFO - Started process 88699 to run task
[2024-06-22T19:20:34.674+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'scheduled__2024-06-22T19:15:02.330472+00:00', '--job-id', '2050', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmp9r9skz_s']
[2024-06-22T19:20:34.675+0000] {standard_task_runner.py:85} INFO - Job 2050: Subtask kafka_data_stream
[2024-06-22T19:20:34.690+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 2050 for task kafka_data_stream (DagRun for kafka_spark_dag with run_id or execution_date of 'scheduled__2024-06-22T19:15:02.330472+00:00' not found; 88699)
[2024-06-22T19:20:34.706+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
